styles.css

```diff
@@ -73,15 +72,174 @@ p {
   margin-top: 10px;
 }
 
+#connects {
+  color: #7f8cff;
+  margin-top: 10px;
+}
+
 #icons {
   height: 20px;
   width: 170.17px;
 }
 
+#sm_icons {
+  height: 20px;
+  width: 170.17px;
+}
+
 #menu {
   justify-items: end;
   height: 12px;
   width: 18px;
   margin-top: 10px;
   margin-left: 155px;
 }
+
+#section1 {
+  display: grid;
+  align-items: center;
+  width: 327px;
+  height: 596px;
+  padding: 16px;
+  gap: 12px;
+  border: #dfe1e6;
+  background-color: white;
+  border-width: 10px;
+  border-radius: 16px;
+}
+
+.section_2 {
+  display: grid;
+  align-items: center;
+  width: 327px;
+  height: 596px;
+  padding: 16px;
+  gap: 12px;
+  border: #dfe1e6;
+  background-color: white;
+  border-width: 10px;
+  border-radius: 16px;
+  margin-top: 50px;
+}
```

Comment:
- [ ] **[OPTIONAL]** Each card in the Figma template has a container, please add it to them. 👇🏽 

- Figma template

![image](https://user-images.githubusercontent.com/58642949/185767078-07bdbefb-a8ec-48ec-8650-a512608e4a31.png)

- Your design 

![image](https://user-images.githubusercontent.com/58642949/185767105-70a00d12-fcda-4a03-82a7-6bd7a8c78733.png)